config/initializers/assets.rb

```diff
@@ -9,3 +9,17 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+
+# For themes. See https://github.com/yoolk/themes_on_rails
+Rails.application.configure do
+  themes_path = "#{Rails.root}/app/themes/"
+  assets_paths = [
+    proc do |path, filename|
+      filename =~ /app\/themes/ && !['.js', '.css'].include?(File.extname(path))
```

Comment:
Maybe I'll add this to themes_on_rails as a Railtie later. Maybe you can PR the changing of Rubocop's rules?